test/ably/resthttp_test.py

```diff
@@ -139,10 +139,8 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
 
+    # RSC7a, RSC7b
```

Comment:
Ah sorry, I don't really mind if it's a comment or docstring, did I miss that before in your PR?  Sorry if I did.  I don't mind which it is, whichever is more idiomatic for Python, so long as the references exist.